### ISSUE TITLE
Demo code - Thumbnails not displaying

### ISSUE THREAD

**Issue by danielaromeo** [chafey/cornerstone#52]
I used the demo code to develop my own Viewer, everything works well, but the thumbnails are not displaying. They're loaded, but using Firebug to inspect the HTML code, I see a canvas with width=0 and height=0 in every .csthumbnail div.
The images are on the same server loaded with absolute path (dicomweb://). The first image in the main viewer is displayed and if I click on the empty thumbnail boxes the images are immediately loaded in the main viewer.
Do you have any thoughts about the cause of this issue?

**Comment by swederik** [comment]
When you enable an element with Cornerstone (https://github.com/chafey/cornerstone/blob/master/src/enable.js) it sets the canvas size to the size of the element (https://github.com/chafey/cornerstone/blob/master/src/resize.js).

If your canvas has width and height of zero it follows that your parent element must have CSS properties that are causing it to have a width and height of zero. Check your CSS for the .csthumbnail divs and try to figure out why they have no dimensions.

**Comment by danielaromeo** [comment]
Thanks for the hint!
I don't know why, but "element.clientWidth" and "element.clientHeight" are equal to 0, but in the css I have "width:100px" and "height:100px". I tried to put also the inline style to the HTML tag, but the result was the same.
Then I tried to transform "element" in a jQuery object and I this is how I modified the code:
`canvas.width = $(element).width();
canvas.height = $(element).height();
canvas.style.width = $(element).width() + "px";
canvas.style.height = $(element).height() + "px";`
Now it works, but I would like to know if this is a bad hack to the code, or if it can be acceptable.
